filesystem_service/src/filesystem_service/monitor.py

```diff
@@ -0,0 +1,169 @@
+"""
+Provides functionality to monitor local filesystem changes
+"""
+
+from multiprocessing import Process
+import os
+
+import inotify.adapters
+
+
+class FileSystemMonitor(object):
```

Comment:
In general, please make sure that the class conforms to the interface declared in the documentation:
https://github.com/epam/OneDrive-L/pull/16

The methods that are not supposed to be public better make private.

Reply:
done